src/include/cephfs/statx.h

```diff
@@ -0,0 +1,79 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * scalable distributed file system
+ *
+ * Copyright (C) 2009-2011 New Dream Network
```

Comment:
Probably need to fix the copyright statement here. We should probably also add a blurb crediting David Howells for the original statx work.